### PR TITLE
Fix: querybuilder updates

### PR DESCRIPTION
**Pull request by switzerb**
Small update for the querybuilder component to ensure classnames are merged in and passed to children. 

From this MR while it still has c2ds embedded in the application context: https://gitlab-cbc2.cce.af.mil/abms/cbc2/jeric2o/jeric2o/-/merge_requests/702

### CODE REVIEW

**Comment by brandonjpierce** [comment]
Changeset needed since this is actual package code change

File: .changeset/gold-bees-smile.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@accelint/design-system": minor
```
Comment (brandonjpierce) [inline .changeset/gold-bees-smile.md:2]: Nit: your changelog message says `patch` but you marked this as a `minor`

File: .changeset/gold-bees-smile.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+patch: fix small bug where classnames didn't propagate
```
Comment (brandonjpierce) [inline .changeset/gold-bees-smile.md:5]: Nit: can make this more human readable / past tense e.g.

```
Fixed bug where classnames didn't propagate for query builder component.
```

File: .changeset/gold-bees-smile.md (hunk 1 above)
Comment (switzerb) [reply .changeset/gold-bees-smile.md:2]: i've actually been a little confused by which language belongs with which change. so a `patch` is a fix but a feature would have a `minor` (or major) label?

**Review by ArrayKnight — DISMISSED** [review]

File: .changeset/gold-bees-smile.md (hunk 1 above)
Comment (brandonjpierce) [reply .changeset/gold-bees-smile.md]: A little more nuanced but realistically the loose definitions provided [here](https://docs.npmjs.com/about-semantic-versioning) can be used as a good rule of thumb.

**Comment by brandonjpierce** [comment]
Notice a minor warning with `rollup` and our ts aliases: 

Screenshot 2024-10-17 at 2 07 13 PM

Potentially okay but figured i'd note it here. Can replicate by running `pnpm run build`

**Comment by brandonjpierce** [comment]
On further investigation the alias warning may indeed have some unfortunate side effects:

https://rollupjs.org/troubleshooting/#warning-treating-module-as-external-dependency

**Comment by switzerb** [comment]
> On further investigation the alias warning may indeed have some unfortunate side effects:
> 
> https://rollupjs.org/troubleshooting/#warning-treating-module-as-external-dependency

looks like maybe the aliasing isn't a good idea right now. i'll revert that bit.

**Comment by brandonjpierce** [comment]
Yeah on the TODO list to redo the build config for design system a bit. It is currently the only package where we have this issue since it needs rollup.

**Pull request merged**